api/ai/priority_algorithm/mutations/greedy_local_max.py

```diff
@@ -31,37 +39,41 @@ def mutate_one(
             priority_team_set
         )
         try:
```

Comment:
github won't let me add a comment to it directly, but if you're going to write a comment for a method, you gotta update that comment when you change the method